src/iqstat.cpp

```diff
@@ -357,3 +227,116 @@ void usage() {
     }
     printReleaseInfo( "iqstat" );
 }
+
+auto show_RuleExec( char *userName,
+                    const char *ruleName,
+                    int allFlag,
+                    bool brief ) -> int
+{
+    int status;
+    genQueryInp_t genQueryInp{};
+    genQueryOut_t *genQueryOut{};
+
+    output_format_parameters output_format;
+
+    auto num_cols_selected = (brief ? output_format.brief_format_len
+                                    : output_format.long_format_len);
+
+    for (int i = 0; i < num_cols_selected; i++) {
+        addInxIval(&genQueryInp.selectInp, output_format.columns[i], 0);
+    }
+
+    char v1[BIG_STR];
+    char v2[BIG_STR];
```

Comment:
Scope could be limited on these since we `strdup` the values in `addInxVal`.

Reply:
Done